core/src/main/scala/org/broadinstitute/dsde/rawls/model/UserAuth.scala

```diff
@@ -25,6 +25,7 @@ case class RawlsGroup(groupName: RawlsGroupName, groupEmail: RawlsGroupEmail, us
 
 object RawlsGroup {
   implicit def toRef(g: RawlsGroup) = RawlsGroupRef(g.groupName)
+  implicit def toRealmRef(g: RawlsGroup) = RawlsRealmRef(g.groupName)
```

Comment:
Good call with the explicit return types. We should probably add a follow-on ticket to update all of our implicits to have return types. I'm sure most of us would appreciate a little bit more speed from IntelliJ at this point...